regression/cbmc-sequentialization/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+  set(gcc_only "-X gcc-only")
```

Comment:
This is incorrect I believe - it should be: `set(gcc_only -X gcc-only)`. 

The difference is that the code as it stands now passes this as a single string, which passed to `test.pl` creates a problem.

For a more detailed explanation, take a look at the commit message here: https://github.com/diffblue/cbmc/pull/5446/commits/741c303064ab837778087c047b8f1df56d65e549

Reply:
Should be fixed now. Sorry I reintroduced this error, it was a result of cut and pasting from before your fix was merged.